[CUBRIDMAN-12] In data comparing, no ignoring to trailing spaces at the end of the data. (#206)
CUBRID/cubrid-manual: en/sql/datatype.rst

    SELECT a FROM t1 WHERE a LIKE 'aaa#%' ESCAPE '#';

Comparison Rules
----------------

> The string values having CHAR-type or VARCAHR-type are compared using one of the two comparison rules as follows:

Reviewer: ```suggestion
When two string values are compared, the following are the comparison rules about the behavior of trailing spaces:
```

*   Comparison for trailing space insensitive
*   Comparison for trailing space sensitive

**Trailing space insensitive**

> If the two string values have fixed length like CHAR-type, the comparison ignores trailing spaces as below example.

Reviewer: ```suggestion
If the two string values are both fixed-length types (CHAR-type), the comparison ignores trailing spaces as below example.
```

Reviewer 2: the original one might cause confusion, instead of like, using '(   )' is better

comparing 'abc ' with 'abc   ' results equal

**Trailing space sensitive**

> If the two string values have variable length like VARCHAR-type, the comparison does not ignore trailing spaces as below example.

Reviewer: ```suggestion
If the two string values are both variable-length types (VARCHAR-type), the comparison does not ignore trailing spaces as below example.
```

comparing 'abc ' with 'abc   ' results 'abc   ' greater than 'abc '

**Exceptions**

> If two string values have each length as fixed and variable, CUBRID follows "trailing space sensitive" rule

Reviewer: ```suggestion
When comparing two string values, If one is a fixed-type and the other is a variable-type, CUBRID follows "trailing space sensitive" rule.
```

> If the configuration parameter value **ignore_trailing_space** is set to **yes**, CUBIRD follows "trailing space insensitive" rule regardless two string values have fixed or variable length.

Reviewer: ```suggestion
If the configuration parameter value **ignore_trailing_space** is set to **yes**, CUBIRD follows "trailing space insensitive" rule regardless of the type of strings.
```
